stock_picking_report_valued/readme/BUGTRACKER.rst

```diff
@@ -0,0 +1,4 @@
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/stock-logistics-reporting/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
```

Comment:
I think this file is not necessary, see https://github.com/OCA/maintainer-tools/tree/master/template/module